### PR TITLE
Add defaultTool to set ToolTip shown

### PR DESCRIPTION
**Pull request by orzyyyy**
<Slider defaultToolTip />

When defaultToolTip is true, Tooltip will be shown always.

[#12743](https://github.com/ant-design/ant-design/issues/12743)

### CODE REVIEW

**Comment by zy410419243** [comment]
So why I pass test units locally but fail on ci?

**Comment by yesmeck** [comment]
Update snapshot by running `npm run test -- -u`, then commit changes.

**Comment by zy410419243** [comment]
@yesmeck I did, but no change in my working tree.  
  
 avatar

File: components/slider/index.en-US.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ To input a value in a range.
 | vertical | If true, the slider will be vertical. | Boolean | false |
 | onAfterChange | Fire when  `onmouseup` is fired. | Function(value) | NOOP |
 | onChange | Callback function that is fired when the user changes the slider's value. | Function(value) | NOOP |
+| defaultToolTip | If true, the Tooltip will be shown always. | Boolean | false |
```
Comment (ztplz) [inline components/slider/index.en-US.md]: If true, the Tooltip will always show.

**Comment by zy410419243** [comment]
what's wrong with the code, test units pass other demos but this one...I have run `npm run test -- -u` before I commit, and the result is in my last comment. 
  
I have tried several times, but snapshot in ci show as below,

 avatar
  
my snapshot is show as below,

 avatar
  
So can anyone help me? I really have no idea of this...

**Pull request closed without merging**